Bench.php

```diff
@@ -311,6 +315,43 @@ public function getLongest()
     }
 
     /**
+     * Pause all marks.
+     *
+     * @return  array
+     */
+    public function pause()
+    {
+        $runningMarks = [];
+
+        foreach ($this as $mark) {
+            if (true === $mark->isRunning() && false === $mark->isPause()) {
```

Comment:
The difference between `isTiming` and `isRunning` is not clear. `isStarted` would be a nicer name for `isRunning`… Any way… let's go for your patch and please open a new issue to address this problem.